core/templates/pages/exploration-editor-page/services/exploration-automatic-text-to-speech.service.spec.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the ExplorationAutomaticTextToSpeechService.
+ */
+
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// the code corresponding to the spec is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
+// ^^^ This block is to be removed.
+
+require('pages/exploration-editor-page/' +
+  'services/exploration-automatic-text-to-speech.service.ts');
+
+describe('Exploration Automatic Text To Speech Service', function() {
+  let eattss = null;
+
+  beforeEach(angular.mock.module('oppia'));
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    var ugs = new UpgradedServices();
+    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
+      $provide.value(key, value);
+    }
+  }));
+
+  beforeEach(angular.mock.inject(function($injector) {
+    eattss = $injector.get('ExplorationAutomaticTextToSpeechService');
+  }));
+
+  it('should test the child object properties', function() {
+    expect(eattss.propertyName).toBe('auto_tts_enabled');
```

Comment:
Use one `it` section to test one method/value (i.e, divide these tests into multiple tests) [Following this [general guideline](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests). @marianazangrossi, should we add this general guide line in the "writing fronted test" wiki page?

Reply:
@simonwex, is this comment resolved?

Also, @marianazangrossi, PTAL I've a question for you?

Reply:
Yeah, I don't think this `it` block is clear. Thanks for the advice, I'll include it in the wiki!

Reply:
@safwan20 I think you should split this `it` block in multiple tests providing a clear description of what is being done.

Reply:
done